CONTRIBUTING.md

```diff
@@ -5,10 +5,11 @@ before contributing is probably sign up for the [LORIS developers' mailing list]
 
 ## Development branches
 
-Pull requests should be based on and sent to the VERSIONNUMBER-dev branch of the version that you are targeting. The master branch is reserved for stable releases.
-
+The master branch is reserved for stable releases.
```

Comment:
There needs to be a blank line between paragraphs in markdown.

Reply:
ok, will adjust.